### PR TITLE
ERC-3000: Optimistic enactment governance standard

### PR DESCRIPTION
**Pull request by izqui**
### [Check spec](https://github.com/aragon/erc3k)

### CODE REVIEW

**Comment by MicahZoltu** [comment]
@izqui Discussed this in EIPIP meeting.  Please add content to this EIP (i.e., get it into draft) ASAP if you want to keep this number.

**Comment by izqui** [comment]
@MicahZoltu alright, will do before the end of the week. Thanks for the notice

**Comment by izqui** [comment]
Ready for review and merging an initial draft @MicahZoltu

File: EIPS/eip-3000.md
Comment (MicahZoltu) [inline EIPS/eip-3000.md:134]:
```suggestion
Configuration specifies the resolver for challenges over scheduled items.
```

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
The discussion-to change is required before merging as a draft.  The rest are either suggestions or may be required changes prior to moving this to Review (the phase after Draft).

**Comment by izqui** [comment]
Thanks for the thorough review @MicahZoltu. All concerns should have been addressed!

File: EIPS/eip-3000.md
Comment (MicahZoltu) [inline EIPS/eip-3000.md:130]:
```suggestion
## Rationale
TBD

## Implementations
```
Rationale is a required section.  It should focus on stating the reason why specific decisions were made over alternate choices.  Some of the content currently is the Motivation would be more appropriate in the Rationale.

**Review by MicahZoltu — APPROVED** [review]
Just needs a Rationale section to be merged as draft.  I still recommend heading the other feedback and getting merged to Review is a bit harder (more strict requirements), but all of that can wait until after Draft is merged except this one.

**Comment by izqui** [comment]
Alright @MicahZoltu, ready to go now!

Thanks for the review, and I will make further changes before asking to change to Review.

**Pull request merged**
